services/vision/client_test.go

```diff
@@ -142,14 +143,18 @@ func TestClient(t *testing.T) {
 		test.That(t, err, test.ShouldBeNil)
 		opts := viscapture.CaptureOptions{
 			true,
-			true, true, true,
+			true,
+			true,
+			true,
 		}
-		capt, err := client.CaptureAllFromCamera(context.Background(), "", opts, map[string]interface{}{})
+		extra := map[string]interface{}{"foo": "captureAll"}
+		capt, err := client.CaptureAllFromCamera(context.Background(), "", opts, extra)
```

Comment:
Can you also add a test to see what happens if you pass "nil" in the extra field?

Reply:
ok, done!